jumpstarter/__init__.py

```diff
@@ -1 +1,4 @@
 __version__ = "0.1.0"
+from .actors import Actor
+
+__all__ = ("Actor",)
```

Comment:
Thanks, I've been meaning to do that but I think we need to also expose the `resource` decorator as well.

Reply:
Agreed. It is very convenient when libraries are structured with public classes/decorators/etc at the root. Something I really need to fix with Sanic at some point.